src/wallet/admin.py

```diff
@@ -42,7 +42,7 @@ def has_delete_permission(self, request, obj=None):
 class WalletTransactionAdmin(admin.ModelAdmin):
     list_display = ('id', 'trx_type', 'amount', 'reference', 'date_created')
     readonly_fields = list_display
-    list_filter = ('trx_type',)
+    # list_filter = ('trx_type',)
```

Comment:
Yes, the commenting out is more of a personal reminder to not forget that we want to have a filter on the transactions status.